mavlink/mavManager.js

```diff
@@ -240,6 +247,28 @@ class mavManager {
     })
   }
 
+  sendHeartbeat (	type = mavheader.MAV_TYPE_ONBOARD_CONTROLLER,
+					autopilot = mavheader.MAV_AUTOPILOT_INVALID,
+					baseMode = 0,
+					customMode = 0,
+					systemStatus = 0,
+					system = this.targetSystem,
```

Comment:
``this.targetSystem`` and ``this.targetComponent`` are the flight controller (the target) not Rpanion. Since Rpanion is (assumed to be) on the vehicle, it makes sense to use ``this.targetSystem``. But we'll need our own component ID, not the flight controller's component ID
I think you meant for the component argument to be passed down to line 269? Then use the default component here as ``minimal.MavComponent.ONBOARD_COMPUTER``

Reply:
Yes, I had changed line 269 to have the component argument passed down but somehow it didn't make it into this commit. Fixed.